src/stories/atoms/card.stories.tsx

```diff
@@ -0,0 +1,172 @@
+import type {Meta, StoryObj} from "@storybook/react";
+
+import {Card} from "../../atoms";
+import {iconLookup} from "../../atoms/icon-store";
+import {CardHeader, IconButton} from "@mui/material";
+
+const meta = {
+  title: "Atoms/Card",
+  component: Card,
+  parameters: {
+    layout: "centered"
+  },
+  tags: ["autodocs"],
+  argTypes: {
+    bgColor: {
+      control: "color",
+      description: "Selects background color"
+    },
+    borderColor: {
+      control: "color",
+      description: "Selects border color"
+    },
+    header: {
+      options: ["Node", "String"],
+      mapping: {
+        Node: (
+          <CardHeader
+            avatar={<iconLookup.business />}
+            action={
+              <IconButton>
+                <iconLookup.add />
+              </IconButton>
+            }
+            title="Header with type node"
+          />
+        ),
+        String: "Header with type string"
+      },
+      description: "Options for the header type"
+    },
+    content: {
+      control: "text",
+      description: "Card content"
+    },
+    footer: {
+      control: "text",
+      description: "Card footer"
+    }
+  },
+  args: {
+    bgColor: "#FFFFFF",
+    borderColor: "#EAECF0",
+    header: "Card Header",
+    content: "Card Content",
+    footer: "Card Footer"
+  }
+} satisfies Meta<typeof Card>;
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+export const WithHeaderTitle: Story = {
```

Comment:
We need to decide as a group if we want to go with "WithHeaderTitle" or "CardWithHeaderTitle" as we have it in button.

@davidc0326 @SGutierrez17

Reply:
Me gusta mas la opción de usar CardWithHeaderTitle

Reply:
Estoy de acuerdo, tiene más intención